Overwrite-Delete FrontEnd

ccelia365 opened the pull request:
FrontEnd for overwrite and delete imported files

ccelia365 committed:
Overwrite-Delete FrontEnd

FrontEnd for overwrite and delete imported files

mjolewis commented:
This can't be committed. This code doesn't do anything. You need to send a post request if someone clicks update or delete.

ccelia365 committed:
BootstrapModal for Overwrite/Delete

Added Bootstrap Modal for Overwrite/Delete Front End and Thymeleaf for Post request

ccelia365 commented:
Updated branch
